sdk/formrecognizer/Azure.AI.FormRecognizer/CustomSerializationSample/CustomSerializationSample.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This project won't be merged - it's just a super simple console app to hold samples.